frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -77,31 +74,33 @@ export const findVMIPod = (
     );
   });
 
-  // Return the newet most ready Pod created
+  // Return the newest, most ready Pod created
   return prefixedPods
     .sort((a: PodKind, b: PodKind) =>
       a.metadata.creationTimestamp > b.metadata.creationTimestamp ? -1 : 1,
     )
     .sort((a: PodKind) => (isPodReady(a) ? -1 : 1))[0];
 };
 
-export const getVMImporterPods = (
+export const getPVCNametoImporterPodsMapForVM = (
   vm: VMKind,
   pods?: PodKind[],
-  pvcNameLabel = `${CDI_KUBEVIRT_IO}/${STORAGE_IMPORT_PVC_NAME}`,
+  pvcs?: PersistentVolumeClaimKind[],
 ) => {
   if (!pods) {
     return null;
   }
 
-  const datavolumeNames = getDataVolumeTemplates(vm)
+  const dataVolumeNames = getDataVolumeTemplates(vm)
```

Comment:
we could create a lookup here as well

Reply:
I only need the names. Are you suggesting I make a lookup and then take the keys from it?

Reply:
ah you are right, maybe creating a `Set` would make more sense then.

Reply:
Good idea! Done!